notebooks/Threshold Optimization Post-Processing for Binary Classification.ipynb

```diff
@@ -12,17 +12,153 @@
    "cell_type": "markdown",
```

Comment:
Are you guys checking in notebooks with outputs?

Reply:
Nope, definitely not! In this particular case someone desperately needed a working version of this, and I didn't want to erase my notebook a few minutes before giving a demo :-)